### PR TITLE
Ammended Product--SearchCatalog-SubCatalog_and_Inventory

### PR DESCRIPTION
**Pull request by nickcoleman**
Amends Product--SearchCatalog-SubCatalog_and_Inventory response
Changes include
- adding items.name (used for item's title)
- identifying sort field keys/direction

### CODE REVIEW

File: Product--SearchCatalog-SubCatalog_and_Inventory.md [hunk 1]
```diff
@@ -56,9 +57,9 @@
     limit: num
     total: num
   },
-  sort: { // provide list + relevance
-    key: <string>,
-    dir: <desc/asc>,
+  sort: {
+    key: 'key', // relevance, title, price, sku_variants, item_id, supplier, last_updated
+    dir: 'dir', // ascending, descending
```
Comment (Gidgidonihah) [inline Product--SearchCatalog-SubCatalog_and_Inventory.md]: @nickcoleman what are we changing on this one? Are we still just passing `asc` or `desc`? Would this be more clear as `<enum(asc,desc)>`?

**Comment by jtprince** [comment]
Oh, and can we call it 'direction' instead of 'dir' (dir is overloaded with directories). @nickcoleman

File: Product--SearchCatalog-SubCatalog_and_Inventory.md (hunk 1 above)
Comment (nickcoleman) [reply Product--SearchCatalog-SubCatalog_and_Inventory.md]: Agree with both ... suggest 

direction: <enum(asc,desc)>

File: Product--SearchCatalog-SubCatalog_and_Inventory.md [hunk 2]
```diff
@@ -56,9 +57,9 @@
     limit: num
     total: num
   },
-  sort: { // provide list + relevance
-    key: <string>,
-    dir: <desc/asc>,
+  sort: {
+    key: 'key', // relevance, title, price, sku_variants, item_id, supplier, last_updated
+    direction: <enum(asc/des)>, // ascending, descending
```
[the remark this answers is not in the document]
Comment (nickcoleman) [reply Product--SearchCatalog-SubCatalog_and_Inventory.md]: sure

**Pull request merged**